subt/ros/robot/launch/ssci-x2.launch

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<launch>
+  <node name="$(arg robot_name)_proxy" pkg="proxy" type="ros_proxy" args="$(arg robot_name)" output="screen">
+    <remap from="$(arg robot_name)/image_raw/compressed" to="$(arg robot_name)/down/image_raw/compressed"/>
+    <remap from="$(arg robot_name)/front/depth" to="$(arg robot_name)/down/depth"/>
+    <remap from="$(arg robot_name)/points" to="$(arg robot_name)/rgbd_camera_down/depth/points"/>
```

Comment:
This surprised me a bit but I am not very good ROSist :wink:. If I grok this right then `remap` is not `->` but instead is `<->` so the order of the arguments does not matter. Is that correct observation?

Reply:
@jisa @pavliik - can you confirm this? Anyway I would proceed and review it again if this shows to be bug.

Reply:
http://wiki.ros.org/roslaunch/XML/remap

The terminology also confuses me. I read it as "when this node looks for $FROM, actually look in the system for $TO". So the terminology seems to be from the perspective of the node and not of the system (which would be more like "make data from $FROM appear in $TO so that $NODE works", but that is not the case).

What worries _me_ is that even after remapping, rosmsg.py still needs to know something about "camera_down".